pkg/kafsar/kafsar.go

```diff
@@ -38,6 +38,10 @@ type KafsarConfig struct {
 	MaxConsumersPerGroup     int
 	GroupMinSessionTimeoutMs int
 	GroupMaxSessionTimeoutMs int
+	NamespacePrefix          string
```

Comment:
I think that need a callback method, not `NamespacePrefix`

Reply:
i will add callback method at next pr